core/eth/state.go

```diff
@@ -39,39 +39,32 @@ func (cs *ChainState) GetOperatorState(ctx context.Context, blockNumber uint, qu
 	}
 
 	return getOperatorState(operatorsByQuorum, quorums, uint32(blockNumber))
-
 }
 
 func (cs *ChainState) GetCurrentBlockNumber() (uint, error) {
-
 	ctx := context.Background()
 	header, err := cs.Client.HeaderByNumber(ctx, nil)
 	if err != nil {
 		return 0, err
 	}
 
 	return uint(header.Number.Uint64()), nil
-
 }
 
-func getOperatorState(operatorsByQuorum [][]core.OperatorStake, quorumIds []core.QuorumID, blockNumber uint32) (*core.OperatorState, error) {
+func getOperatorState(operatorsByQuorum core.OperatorStakes, quorumIds []core.QuorumID, blockNumber uint32) (*core.OperatorState, error) {
 	operators := make(map[core.QuorumID]map[core.OperatorID]*core.OperatorInfo)
 	totals := make(map[core.QuorumID]*core.OperatorInfo)
 
 	for i, quorum := range operatorsByQuorum {
```

Comment:
Replace `i` with `quorumID` here, since `i` is actually the quorumID.

Reply:
Nice catch!